packages/wua_windows.go

```diff
@@ -27,38 +27,41 @@ var wuaSession sync.Mutex
 
 // IUpdateSession is a an IUpdateSession.
 type IUpdateSession struct {
-	con *ole.Connection
-	ses *ole.Dispatch
+	*ole.IDispatch
 }
 
 func NewUpdateSession() (*IUpdateSession, error) {
 	wuaSession.Lock()
 	if err := ole.CoInitializeEx(0, ole.COINIT_MULTITHREADED); err != nil {
-		wuaSession.Unlock()
-		return nil, err
+		e, ok := err.(*ole.OleError)
+		// S_OK = 0, S_FALSE = 1, both are Success codes.
```

Comment:
Are these available as constants somewhere? If not, maybe make them consts here in favor of self-documented code.